packages/main/src/Input.js

```diff
@@ -831,6 +831,10 @@ class Input extends UI5Element {
 		};
 	}
 
+	get itemSelectionAnnounce() {
+		return this.Suggestions.selectedItemIndex !== null ? this.Suggestions.itemSelectionAnnounce : undefined;
```

Comment:
I think you have to check if `this.Suggestions` is present, because it is undefined if "showSuggestions" is false.